superset/config.py

```diff
@@ -1092,6 +1092,9 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
 # It will get executed each time when user open a chart's explore view.
 DATASET_HEALTH_CHECK = None
 
+# SQLAlchema link doc reference
+SQLALCHEMY_DOCS_URL = "https://docs.sqlalchemy.org/en/rel_1_2/core/engines.html#"
```

Comment:
tiny tiny nit, but prob don't need the # at the end.
I suppose we should also update the link to `https://docs.sqlalchemy.org/en/13/core/engines.html` since we're on 1.3 now.